Tasks/Common/AzureRmDeploy-common/azure-arm-rest/azure-arm-app-service-kudu.ts

```diff
@@ -455,7 +455,7 @@ export class Kudu {
         httpRequest.body = fs.createReadStream(webPackage);
 
         try {
-            let response = await this._client.beginRequest(httpRequest, null, 'application/octet-stream');
+            let response = await this._client.beginRequest(httpRequest, null, 'multipart/form-data');
```

Comment:
Why is this change required?

Reply:
According to this document https://docs.microsoft.com/en-us/azure/app-service/deploy-zip multipart/form is the recommended content-type for both zip and war deploy.

Reply:
@vincentdass - I recollect you had changed the content type earlier to octet-stream for fixing an issue. Can you review this to see we are not regressing anything now.